psi4/extras.py

```diff
@@ -204,3 +204,23 @@ def test(extent='full', extras=None):
 
     retcode = pytest.main(command)
     return retcode
+
+def error_box(message: str = None):
+    """ put a message string into a box for extra attention
+
+    Parameters
+    -----------
+    message
+       message string to be boxed
+
+    Returns
+    --------
+    str
+       box containing the message as a multiline string
+    """
```

Comment:
I think we need something like:

```python
box_width = 120
message = " ".join(message.split("\n"))
from textwrap import wrap
message = wrap(message, box_width, break_long_words=False)

error_str = []
error_str.append('\n!' + '-' * len(box_width) + ' !\n')

fmt = "{:" + str(box_width) + "}"
for line in message:
    error_str.append(fmt.format(line))
...

```

This will handle long lines gracefully and will correctly pad the width like the rest of the code. @loriab might have a box style built in.

Reply:
there are the banners https://github.com/psi4/psi4/blob/master/psi4/driver/p4util/text.py#L157 but they're pretty old and could probably benefit from `wrap` themselves.

Reply:
Good point. I assumed only a short, single line error message needs to be boxed.